validator/client/src/main/java/tech/pegasys/teku/validator/client/duties/BlockProductionDuty.java

```diff
@@ -100,8 +103,12 @@ private SafeFuture<BLSSignature> createRandaoReveal(final ForkInfo forkInfo) {
 
   private SafeFuture<Optional<BlockContainer>> createUnsignedBlock(
       final BLSSignature randaoReveal) {
-    return validatorApiChannel.createUnsignedBlock(
-        slot, randaoReveal, validator.getGraffiti(), useBlindedBlock);
+    if (this.blockV3Enabled) {
```

Comment:
could remove `this.` if it's not an assignment

Reply:
Done